ietf-dtnma-agent.yang

```diff
@@ -61,6 +61,27 @@ module ietf-dtnma-agent {
     }
   }
 
+  grouping object-id {
+    description
+      "Common parameters for providing object id";
+    amm:parameter namespace {
+      description
+        "Namespace";
+      amm:default "//ietf/dtnma-agent/";
+      amm:type "/ARITYPE/TEXTSTR";
+    }
+    amm:parameter obj-name {
+      description
+        "Text obj ID";
+      amm:type "/ARITYPE/TEXTSTR";
```

Comment:
Per [AMM Section 3.4](https://www.ietf.org/archive/id/draft-ietf-dtn-amm-04.html#section-3.4.1) this should be restricted to a limited valid text pattern.